Sources/App/Views/Sections/Editor/SourceTree/SourceTree.swift

```diff
@@ -35,7 +35,7 @@ struct SourceTree: View {
             ) {}
           )
       } else {
-        List(viewModel.filteredNodes, children: \.children) { node in
+        List(viewModel.sourceTree.children ?? [], children: \.children) { node in
```

Comment:
Why did you removed filteredNodes from here?
```suggestion
        List(viewModel.filteredNodes, children: \.children) { node in
```

Reply:
because we are not yet using the filtered nodes